### PR TITLE
Task#4280 Create UI Genres

### PR DESCRIPTION
**Pull request by vuanhmai2233**
Link redmine : https://edu-redmine.framgia.vn/issues/4280 .
Description : Create UI Tab Genres . 
photofacefun_com_1545752066

### CODE REVIEW

**Comment by vuanhmai2233** [comment]
ready

File: app/src/main/java/com/framgia/music_44/screen/genres/GenresFragment.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import com.framgia.music_44.data.model.Genre;
+import com.framgia.music_44.screen.genres.adapter.GenresAdapter;
+import com.framgia.music_44.util.Genres;
+import com.framgia.music_44.util.Navigator;
+import com.framgia.music_44.util.OnItemClickListener;
+import java.util.ArrayList;
+import java.util.List;
+
+public class GenresFragment extends Fragment implements OnItemClickListener {
+    private Navigator  mNavigator;
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/framgia/music_44/screen/genres/GenresFragment.java]: reformat code

File: app/src/main/java/com/framgia/music_44/screen/home/adapter/HomeAdapter.java [hunk 2]
```diff
@@ -30,7 +30,7 @@ public HomeAdapter(Context context, OnItemClickListener listener) {
 
     public void addSongs(List<Songs> songs) {
         mSongs.addAll(songs);
-        notifyItemRangeChanged(mSongs.size() - ONE, songs.size() - ONE);
+        notifyItemRangeInserted(mSongs.size()-songs.size(),songs.size());
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/framgia/music_44/screen/home/adapter/HomeAdapter.java]: reformat code

File: app/src/main/res/layout/item_genres.xml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    android:layout_marginVertical="@dimen/dp_5"
+    >
+    <android.support.v7.widget.CardView
+        android:scrollbarStyle="insideInset"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="@dimen/dp_20"
+        app:cardBackgroundColor="#ebc5f7"
+        app:cardCornerRadius="20dp"
+        app:cardElevation="20dp"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/item_genres.xml]: hard code

File: app/src/main/res/layout/item_genres.xml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    android:layout_height="wrap_content"
+    android:layout_marginVertical="@dimen/dp_5"
+    >
+    <android.support.v7.widget.CardView
+        android:scrollbarStyle="insideInset"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="@dimen/dp_20"
+        app:cardBackgroundColor="#ebc5f7"
+        app:cardCornerRadius="20dp"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/item_genres.xml]: hard code

File: app/src/main/res/layout/item_genres.xml [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+        app:cardElevation="20dp"
+        >
+        <TextView
+            android:textStyle="bold"
+            android:id="@+id/textViewGenres"
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/dp_80"
+            android:gravity="center"
+            android:textColor="#ffffff"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/item_genres.xml]: hard code

**Comment by vuanhmai2233** [comment]
ready

File: app/src/main/res/values/colors.xml [hunk 6]
```diff
@@ -11,4 +11,6 @@
     <color name="color_tim">#482B8F</color>
     <color name="color_dark1">#000000</color>
     <color name="color_dark2">#25232b</color>
+    <color name="color_timm">#af6eef</color>
+    <color name="color_hong">#ebc5f7</color>
```
Comment (tuanlvt-1117) [inline app/src/main/res/values/colors.xml]: (tat2) đặt tên chung 1 ngôn ngữ, nên default là EL . không dùng vietnamese

**Comment by vuanhmai2233** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**